app/models/blog.rb

```diff
@@ -10,7 +10,7 @@ class Blog < ApplicationRecord
   scope :published, -> { where('secret = FALSE') }
 
   scope :search, lambda { |term|
-    where("title LIKE '%#{term}%' OR content LIKE '%#{term}%'")
+    where('title LIKE ? OR content LIKE ?', "%#{term}%", "%#{term}%")
```

Comment:
```suggestion
    where('title LIKE :term OR content LIKE :term', term: "%#{term}%")
```

って書くと少しDRYにできます。

あと、これは必須ではないですが、like文のエスケープ処理というのもあるので調べてみてください〜。

Reply:
>like文のエスケープ処理というのもあるので調べてみてください〜。

調べてみました。`sanitize_sql_like`というものがあったのですね。
`sanitize_sql_like`を使うようにしました。